test/exec-capabilityboundingset-invert.service

```diff
@@ -0,0 +1,6 @@
+[Unit]
+Description=Test for CapabilityBoundingSet
+
+[Service]
+ExecStart=/bin/sh -c 'c=$(capsh --print | grep "Bounding set " | cut -f 2 -d "="); echo $c; exit $(test -n $c)'
+CapabilityBoundingSet=~
```

Comment:
This is a no-op test. Test passes without `CapabilityBoundingSet=~` too. Maybe
```
CapabilityBoundingSet=
CapabilityBoundingSet=~
```
?

Also, I think, we should compare: `pcaps ppid` == `pcaps pid`.
Something like:
```
ExecStart=/bin/bash -c 'parent_caps=$(getpcaps $PPID |& cut -d= -f2); my_caps=$(getpcaps $$$$ |& cut -d= -f2); [[ "$parent_caps" = "$my_caps" ]]'
```